src/compiler/data.ts

```diff
@@ -18,7 +19,7 @@ import {NOMINAL, ORDINAL, QUANTITATIVE, TEMPORAL} from '../type';
  *                 If the encoding contains aggregate value, this will also create
  *                 aggregate table as well.
  */
-export default function(model: Model) {
+export function def(model: Model) {
```

Comment:
While default is in the ES6 standard, that doesn't mean that it is a good practice.   

By using default, that means you always create alias with arbitrary names every time you import and there is no mechanism for enforcing that you will name the alias similarly.  Therefore, unless there is a good reason.  Providing method name such as `compileData()` and do `import {compileData} from './data.ts'` is a much better way.   On that note, I'm creating a PR to change all confusing `def()`, `defs()` to `compileXXX()` ... will send you when I'm done.